guides/source/configuring.md

```diff
@@ -662,6 +662,17 @@ Enables the use of SHA256 fingerprints in asset names. Set to `true` by default.
 
 Disables the concatenation and compression of assets. Set to `true` by default in `development.rb`.
 
+In previous versions of Rails, `config.assets.debug` was set to `true` by default in the development environment, which disabled the concatenation and preprocessing of assets to facilitate easier debugging. This was particularly useful when working with a large number of complex assets, as it allowed developers to trace issues directly to specific files.
+
+However, as of Rails 7.0 (commit adec7e7 on Aug 10, 2021), this setting is no longer included by default in the `development.rb` configuration. The change was made because the move towards using ECMAScript Modules (ESM) in development generally eliminates the need for concatenating or preprocessing large JavaScript files, rendering debug mode less relevant.
+
+To explicitly enable or disable asset debugging in the current version of Rails, developers may add the following line to `config/environments/development.rb`:
+
+```ruby
+# Enable or disable asset concatenation and preprocessing for easier debugging
+config.assets.debug = true  # Set to `false` to enable concatenation and compression
+```
```

Comment:
@devikasugathan Thanks for the PR,

Actually we don't really need to mention the history of a configuration file. This document only refers what the purpose of `config.assets.debug` when it set to true.

also you can add following pattern which we are already using that for other configuration

```
The default value depends on the `config.load_defaults` target version:

| Starting with version | The default value is |
| --------------------- | -------------------- |
| (original)            | `true`               |
| 7.0                   | `false`              |
```

Reply:
I agree with @akhilgkrishnan, it would be nice to use a table here.
We could use that for similar config as well.
Maybe something like:
```suggestion
| Starting with version | The value in development.rb |
| --------------------- | -------------------- |
| (original)            | `true`               |
| 7.0                   |  Not set           |
```

The whole history of the setting is very thorough.
It's probably too much detail for this guide which is already very large.
Maybe we can link to the original commit instead?

```suggestion
| Starting with version | The value in development.rb |
| --------------------- | -------------------- |
| (original)            | `true`               |
| 7.0                   |  [Removed](https://github.com/rails/rails/commit/adec7e7ba87e3d268149d0020d54dc211eb02ada)           |
```